metrics/ddl.go

```diff
@@ -31,7 +31,7 @@ var (
 			Subsystem: "ddl",
 			Name:      "handle_job_duration_seconds",
 			Help:      "Bucketed histogram of processing time (s) of handle jobs",
-			Buckets:   prometheus.ExponentialBuckets(0.01, 2, 20),
+			Buckets:   prometheus.ExponentialBuckets(0.01, 2, 22), // 10ms ~ 12hours
```

Comment:
Do we unify the expression？Some place we use `4096s`  like line91.

Reply:
I think no need to do it. 4096 is also easy to understand. And the range of this value is not accurate.

Reply:
Got it.

Reply:
could add more comments about how these ranges are calculated?

Reply:
For this one only or all of them?